File: dev/test_data/python/test_main/aas_core_meta.v3/input/snippets/Verification/is_xs_date_time_UTC.py
def is_xs_date_time_utc(value: str) -> bool:
    """
    Check that :paramref:`value` is a ``xs:dateTime`` with
    the time zone set to UTC.
    """
    if not matches_xs_date_time_utc(value):
        return False

    date, _ = value.split("T")
    return is_xs_date(date)

File: dev/test_data/test_main/typescript/aas_core_meta.v3/input/snippets/Verification/is_xs_date_time_UTC.ts
/**
 * Check that `value` is a valid `xs:dateTime` with
 * the time zone set to UTC.
 *
 * @param value - to be checked
 * @returns `true` if `value` is a valid `xs:dateTime` with the UTC time zone
 */
export function isXsDateTimeUtc(
  value: string
): boolean {
  if (!matchesXsDateTimeUtc(value)) {
    return false;
  }

  const date = value.split("T", 1)[0];
  return isXsDate(date);
}
